doc/DHCPv6_relay/DHCPv6-relay-agent-High-Level-Design.md

```diff
@@ -165,7 +165,7 @@ RELAY-REPLY
 <pre>
 DHCP|intf-i|dhcpv6_servers: [&quot;dhcp-server-0&quot;, &quot;dhcp-server-1&quot;, ...., &quot;dhcp-server-n-1&quot;]
 
-DHCP|intf-i|dhcpv6_option|link_layer_addr: [&quot;79&quot;]
+DHCP|intf-i|dhcpv6_option|link_layer_addr: &quot;true&quot;
```

Comment:
what about naming this option rfc6939_support?

Reply:
Updated!